src/user-store.js

```diff
@@ -194,6 +194,20 @@ class UserStore {
       })
       .then(() => this.backend.put('users', userKey, user))
   }
+  /**
+   * Permanently deletes the files of a user
+   *
+   * @param user {UserAccount}
+   *
+   * @return {Promise}
+   */
+
+  deleteUser (user) {
+    let userKey = UserStore.normalizeIdKey(user.id)
+
+    return Promise.resolve()
+      .then(() => this.backend.del('users', userKey))
```

Comment:
I would prefer `this.backend.remove` as that is what in exposed in the documentation for https://github.com/solid/kvplus-files, but I see that there is an alias with `del` there, so I'm fine with it.

Reply:
Just got to know that `del` is recommended by @dmitrizagidulin , so ignore the comment above ^_^